Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -331,26 +296,14 @@ void Renderer::DrawDebugText()
 		std::string res_text;
 		switch (g_ActiveConfig.iEFBScale)
 		{
-		case SCALE_AUTO:
-			res_text = "Auto (fractional)";
-			break;
 		case SCALE_AUTO_INTEGRAL:
 			res_text = "Auto (integral)";
```

Comment:
If we won't support fractional scaling avy more, than there is no need to be explicit here:
res_text = "Auto";

Also, I think we should rename the enum SCALE_AUTO_INTEGRAL to SCALE_AUTO.